src/frontend/js/components/DashboardModerators/ButtonChangeRoleCta/index.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import 'regenerator-runtime/runtime';
```

Comment:
```suggestion
```

this should not be done here. You just fix the use of regenerator on this test. But in fact regenerator must be used globally and not only for tests because babel will need it to transpile to the targets you defined.
2 things you have to do:

- add `'regenerator-runtime/runtime'` in your webpack entry section
- add `'regenerator-runtime/runtime'` in your jest `setupFilesAfterEnv` section

Also, most of time the same thing is made when you use core-js

Reply:
I included it in index.tsx, webpack is indeed a much nicer way, thank you for the feedback